WeatherMonitoringSystem/Domain/Bots/RainBot.cs

```diff
@@ -0,0 +1,25 @@
+﻿
+using WeatherMonitoringSystem.Config;
+using WeatherMonitoringSystem.Domain.Config;
+
+namespace WeatherMonitoringSystem.Bots
+{
+    public class RainBot : IWeatherBot
+    {
+        private readonly BotConfig _config;
+
+        public RainBot()
+        {
+            _config = ConfigurationManager.Instance.RainBot;
+        }
+
+        public string Activate(double temperature, double humidity)
```

Comment:
when activating the bot, it would make sense to return a bool to indicate whether the bot is activated or not, and leave the print statements to the caller. Same comment to the other bots.

Reply:
we would also need to save a bool in this class to indicate whether the bot is activated or not